.gitignore

```diff
@@ -0,0 +1,4 @@
+pyproject.toml
+.venv/
+.vscode/
+app/__pycache__/
```

Comment:
There is a whole `.gitignore` generators in the Internet. Try to stick to a more comprehensive version of it.